cpp/src/parquet/CMakeLists.txt

```diff
@@ -159,6 +159,8 @@ set(PARQUET_SRCS
     arrow/variant_internal.cc
     arrow/writer.cc
     bloom_filter.cc
+    bloom_filter_builder_internal.cc
+    bloom_filter_writer_internal.cc
```

Comment:
nit: should we merge these two files into one?

Reply:
Personally, `bloom_filter_writer_internal.cc` is only used in `column_writer.cc`, and `bloom_filter_writer_internal.cc` is more widely used?